ourhexenv.py

```diff
@@ -192,7 +194,6 @@ def step(self, action):
             self.place_piece(row, col, marker)
 
             if self.check_winner(marker):
-                self.terminations = {agent: True for agent in self.agents}
```

Comment:
Why did we delete this line? We need to terminate the episode for both agents if either of them win.

Reply:
Seems to have been removed while resolving conflict. Adding it back.